x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -98,45 +99,6 @@ export async function getPackageSavedObjects(
 
 export const getInstallations = getPackageSavedObjects;
 
-export async function getPackageInfoFromRegistry(options: {
-  savedObjectsClient: SavedObjectsClientContract;
-  pkgName: string;
-  pkgVersion: string;
-}): Promise<PackageInfo> {
```

Comment:
@nchaulet could you double check how I consolidated these two functions?

Reply:
I need to test it locally but I think it should work correctly. using the registry by default for every case and fallback to the installed package if the registry did not work 👍